firmware/framebuffer.h

```diff
@@ -22,16 +22,17 @@
  * Frame buffers must be aligned to XXX boundary.
  *
  *  0x01000000 - Pattern Buffer - Frame Buffer n
+ *  0x04000000 - PCIe Buffer - Frame Buffer n
  *
  * Each input then has 3 frame buffers spaced like this;
  *  // HDMI Input 0
- *  0x02000000 - HDMI Input 0 - Frame Buffer n
- *  0x02040000 - HDMI Input 0 - Frame Buffer n+1
- *  0x02080000 - HDMI Input 0 - Frame Buffer n+2
+ *  0x05000000 - HDMI Input 0 - Frame Buffer n
```

Comment:
Why have these moved?

Reply:
Input buffers and PCIe buffer were moved so that PCIe-Wishbone bridge can access the data inside them.
Bridge can access 32M of memory starting at 0x44000000.